plugins/newsdownloader.koplugin/main.lua

```diff
@@ -66,15 +66,55 @@ function NewsDownloader:init()
     self.ui.menu:registerToMainMenu(self)
 end
 
+local function getWifiEnableAction()
```

Comment:
Is there a problem with making this generic? This function looks decent but it should part of NetworkManager.

Reply:
Might be hard to find it there but ok. 

Hopefully someone will reorganize with #3602..

Reply:
I'm afraid I don't understand what you're trying to say. This PR has nothing to with NewsDownloader. Other than to remove the earlier hack it shouldn't be touched at all.